src/views/new-evaluation/new-evaluation.js

```diff
@@ -702,6 +113,6 @@ function NewEvaluationForm (options = {}) {
 }
 
 window.onload = function () {
-    const appEl = document.querySelector("#app");
+    const appEl = document.querySelector('#app');
```

Comment:
Why did you choose `querySelector` here? Are there other, possibly better options?